rts/lra-examples/pom.xml

```diff
@@ -44,5 +44,6 @@
   <modules>
     <module>lra-coordinator</module>
     <module>cdi-participant</module>
+    <module>lra-jwt</module>
```

Comment:
Do you have any recommendation, where could I add this quickstart to?

Reply:
I was re-revisiting my comment here and I assume the structure of the `rts` folder is a bit strange. So please, leave the quickstart where it is but please update the `rts/README.md` where will be more information what can be found in the folder (ie. `at`, `lra`, `lra-examples`).
It's not directly connected with this issue but it could not make much effort to add two bullets and short paragraph and it will make all clearer.

Reply:
okay